pkg/pillar/utils/logging/log.go

```diff
@@ -32,3 +34,33 @@ func GetMyStack() string {
 	}
 	return output
 }
+
+// NoSuitableAddrStr - the string to be used for checking the error message
+const NoSuitableAddrStr string = "no suitable address found"
+
+// NoSuitableAddrErr - the function to check 'no suitable address' for http mainly
+func NoSuitableAddrErr(err error) bool {
```

Comment:
Nit, but shouldn't this function be called IsNoSuitableAddrErr as in IsEOF, IsTemporary and other predicates on errors?